### ISSUE TITLE
Audio Stutter and poor quality audio in Xak III

### ISSUE THREAD

**Issue by amerika13** [Mazamars312/openfpga-pcengine-cd#35]
I was attempting to play my English patched Xak III on my Pocket when I went down a rabbit hole. My SD card was originally formatted as Fat32 but then I read that the driver is slow and we aren't sure when it will be fixed so the best thing to do was go ExFAT and 32k cluster size. Done.

I fire up Xak III and a few things happen.

1. The redbook audio track that plays in the beginning stutters like crazy
2. The Core menu, upon opening, is EXTREMELY laggy when a music file is attempting to play. When one is not playing, it runs fine.
3. Adjusting the Audio Timer did not help (going in either direction). I had to adjust this on the BIOS screen since it wouldn't take a long time to do.
4. Going into the game where you can control your character, the quality of music is quite poor and does not match what it should sound like (regardless of Audio Timer settings).

I then repeated all of this with the original copy ripped identically to Redumps methods. I then used CDMage to merge the bin files as there were 35 of them (the readme said you could only have max 27 for now). I then found out that the original JP version does the same thing. So the issues have nothing to do with the English patch.

I then tried a few other common games and found even Lords of Thunder suffered from really poor sound quality.

So, as others have written about, there is an issue. Hopefully my write-up gives a few more repro steps or at least a potential test for a future version of the core.

Thank you again for writing this core! You are amazing and the core will be absolutely fantastic when it's finished!!!!!! I am very excited.

**Comment by amerika13** [comment]
I just updated to the latest Pocket Firmware and it appears that Xak III no longer stutters with the music played at the start of the game before you choose Continue. However, the audio quality itself is still not good. I reset the core to "defaults" to make sure it wasn't a setting I had changed causing the issue..

**Comment by amerika13** [comment]
I know this core isn't being actively updated, but I did try the pocket_firmware_1_2.bin dated for Dec 7th, 2023 and no change.

**Comment by amerika13** [comment]
I tried 0.16 to see if I could get things to sound better. And I couldn't get the game to launch. However, after updating again to 0.17, I changed the audio timing again (I'd done this before) and this time it seemed to do something. I set it to the lowest value and now the game sounds pretty good overall. No clue if it's accurate, but to my ear it sounds good.

So I'll go ahead and close this out and thank you for your work on this.
